src/editor.js

```diff
@@ -81,6 +81,15 @@ class TextEditorEmbed {
 		return this
 	}
 
+	async save() {
+		return this.#vm.$children[0].save?.()
```

Comment:
Maybe we could move `this.#vm.$children[0]` to a private getEditor method for more readability?